strawberryfield.module

```diff
@@ -25,12 +25,38 @@ use Drupal\Core\Field\FieldItemListInterface;
  */
 function strawberryfield_node_presave(ContentEntityInterface $entity) {
 
+
   if ($sbf_fields = \Drupal::service('strawberryfield.utility')->bearsStrawberryfield($entity)) {
+    $config = \Drupal::config('strawberryfield.general');
+    $bench = FALSE;
+    // When benchmark is enabled a simple but effective report will be found in the reports/logs
+    if ($config->get('benchmark')) {
+      $bench = TRUE;
+    }
+    // Introducing our newest development, the processing time stats!
+    $start_time = microtime(true);
+
     $event_type = StrawberryfieldEventType::PRESAVE;
     $event = new StrawberryfieldCrudEvent($event_type, $entity, $sbf_fields);
     /** @var \Symfony\Component\EventDispatcher\EventDispatcher $dispatcher */
     $dispatcher = \Drupal::service('event_dispatcher');
     $dispatcher->dispatch($event_type, $event);
+
+    if ($bench) {
+      $end_time = microtime(TRUE);
```

Comment:
I want not only to track this as a log, but inject into some setting/entity/even a queue? i can then process/avg?
pretty sure this processing is not even expensive. I was surprised really by how fast the filesystem was, specially S3! but probably the save event and the update events, because of MYSQL can be slower.